kubernetes/samples/scripts/create-rcu-credentials/README.md

```diff
@@ -0,0 +1,60 @@
+# Creating RCU credentials for a Fusion Middleware domain
+
+This sample demonstrates how to create a Kubernetes secret containing the
+RCU credentials for a Fusion Middleware domain.  The operator expects this secret to be
+named following the pattern `domainUID-rcu-credentials`, where `domainUID`
+is the unique identifier of the domain.  It must be in the same namespace
+that the domain will run in.
+
+To use the sample, run the command:
+
+```
+$ ./create-rcu-credentials.sh \
+  -u username \
+  -p password \
+  -a sys_username \
+  -q sys_password \
+  -d domainUID \
+  -n namespace \
+  -s secretName
+```
+
+The parameters are as follows:
+
+```  
+  -u username for schema owner (regular user), must be specified.
+  -p password for schema owner (regular user), must be specified.
+  -a username for SYSDBA user, must be specified.
+  -q password for SYSDBA user, must be specified.
+  -d domainUID, optional. The default value is domain1. If specified, the secret will be labeled with the domainUID unless the given value is an empty string.
+  -n namespace, optional. Use the default namespace if not specified.
+  -s secretName, optional. If not specified, the secret name will be determined based on the domainUID value.
+```
+
+This creates a `generic` secret containing the user name and password as literal values.
+
+You can check the secret with the `kubectl get secret` command.  An example is shown below,
+including the output:
+
+```
+$ kubectl -n domain-namespace-1 get secret domain1-rcu-credentials -o yaml
+apiVersion: v1
+data:
+  password: d2VsY29tZTE=
+  username: d2VibG9naWM=
+  sys_username: c3lz
+  username: c29hMQ==
```

Comment:
Based on the script below the keys would be `username`,  `password`, `sys_username` and `sys_password` so this output does not appear correct with 2 `username` keys ???

It might be easier to use output of `kubectl describe` but that just may be me since it does not show the data

```
$ kubectl describe -n domain1-ns secret domain1-weblogic-credentials
Name:         domain1-weblogic-credentials
Namespace:    domain1-ns
Labels:       weblogic.domainName=domain1
              weblogic.domainUID=domain1
Annotations:  <none>

Type:  Opaque

Data
====
password:  8 bytes
username:  8 bytes
```

Reply:
fixed